### DISCUSSION TITLE
From where we can download the latest Container image for keycloak

### DISCUSSION THREAD

**Discussion by patil23887** [keycloak/keycloak#16904] [Q&A] [answered]
currently we are using jboss/keycloak - 16.1.1 image from docker hub but in this image there is many high and critical CVE are there and no any latest version are there in docker hub. so can you please help us from where we have to download the latest container image of keycloak

below are CVE in jboss/keycloak 16.1.1 version so we require new latest image of keycloak in which this CVE are fixed 
https://nvd.nist.gov/vuln/detail/CVE-2022-46364
https://nvd.nist.gov/vuln/detail/CVE-2022-1245
https://nvd.nist.gov/vuln/detail/CVE-2022-42004
https://nvd.nist.gov/vuln/detail/CVE-2022-42003
https://nvd.nist.gov/vuln/detail/CVE-2022-42003
https://nvd.nist.gov/vuln/detail/CVE-2022-41881
https://nvd.nist.gov/vuln/detail/CVE-2022-25857
https://nvd.nist.gov/vuln/detail/CVE-2022-25844
https://nvd.nist.gov/vuln/detail/CVE-2022-23913
https://nvd.nist.gov/vuln/detail/CVE-2022-21476
https://nvd.nist.gov/vuln/detail/CVE-2022-2053
https://www.cve.org/CVERecord?id=CVE-2022-1319
https://nvd.nist.gov/vuln/detail/CVE-2022-1259
https://nvd.nist.gov/vuln/detail/CVE-2021-3859
https://nvd.nist.gov/vuln/detail/CVE-2020-36518
https://nvd.nist.gov/vuln/detail/CVE-2020-36518
https://nvd.nist.gov/vuln/detail/CVE-2020-36518
https://nvd.nist.gov/vuln/detail/CVE-2019-20916
https://github.com/hornetq/hornetq/commit/c076c18174d67c08906ff88034b204cbf17c6efe
https://nvd.nist.gov/vuln/detail/CVE-2022-3510
https://nvd.nist.gov/vuln/detail/CVE-2022-3509
https://nvd.nist.gov/vuln/detail/CVE-2022-3143
https://nvd.nist.gov/vuln/detail/CVE-2022-42003
https://nvd.nist.gov/vuln/detail/CVE-2022-42004
https://nvd.nist.gov/vuln/detail/CVE-2022-23913
https://nvd.nist.gov/vuln/detail/CVE-2022-42004
https://nvd.nist.gov/vuln/detail/CVE-2022-45047
https://nvd.nist.gov/vuln/detail/CVE-2022-1245
https://nvd.nist.gov/vuln/detail/CVE-2022-0839
https://github.com/pypa/pip/issues/8606
https://github.com/square/okhttp/issues/6738
https://issues.apache.org/jira/browse/LUCENE-9981
https://nvd.nist.gov/vuln/detail/CVE-2022-46363
https://nvd.nist.gov/vuln/detail/CVE-2022-31197
https://nvd.nist.gov/vuln/detail/CVE-2022-21724
https://nvd.nist.gov/vuln/detail/CVE-2021-3827
https://nvd.nist.gov/vuln/detail/CVE-2020-13692

**Comment by sirishkumar** [comment, marked as answer]
Its mentioned here https://www.keycloak.org/downloads
